### DISCUSSION TITLE
"Not existing property cc for go\modules\community\tasks\model\Task" when trying to create a task

### DISCUSSION THREAD

**Discussion by awnz** [Intermesh/groupoffice#1062] [Q&A] [answered]
Hi all,

When testing Tasks, if I try to create a task, I get the error message "Not existing property cc for go\modules\community\tasks\model\Task".
"CC" is not a shown field when creating a task. 

This is what's being sent (from Firefox 118.0.2 on Ubuntu 22.04):
```
[["Task/set",{"create":{"ext-gen944":{"title":"Test 3","cc":"","start":null,"due":null,"estimatedDuration":0,"progress":"needs-action","percentComplete":0,"priority":0,"tasklistId":9,"responsibleUserId":null,"categories":[],"description":"","location":"","alerts":{}}},"ifInState":"5:0:0"},"clientCallId-26"]]
```

If I use the browser debug console to strip `"cc":"",` out of the form data and resend it, the task is successfully created.

Should "cc" be in there?

**Comment by mschering** [comment, marked as answer]
Sorry for the inconvenience. It slipped into 6.8.15. This issue has been fixed in 6.8.16

**Reply by awnz** [reply]
Cheers :)

**Reply by awnz** [reply]
Sorry forgot to confirm - fix verified 🙂
